### PR TITLE
[doc] add 3rd party libs integrations for expo-sqlite/next

### PR DESCRIPTION
**Pull request by Kudo**
# Why

add 3rd party integrations like Drizzle ORM

# How

add a section for 3rd party libs integrations and add Drizzle ORM and Knex.js

# Test Plan

ci passed

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/sqlite-next.mdx [hunk 1]
```diff
@@ -343,6 +343,23 @@ const row = await db.getFirstAsync<{ data: Uint8Array }>('SELECT * FROM blobs');
 expect(row.data).toEqual(blob);
 ```
 
+## Third-party libraries integrations
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/sqlite-next.mdx:346]:
```suggestion
## Third-party library integrations
```

File: docs/pages/versions/unversioned/sdk/sqlite-next.mdx [hunk 2]
```diff
@@ -343,6 +343,23 @@ const row = await db.getFirstAsync<{ data: Uint8Array }>('SELECT * FROM blobs');
...
+
+The `expo-sqlite` library is designed to be a solid SQLite foundation. It enables broader integrations with third-party libraries for more advanced features. Here are some of the libraries that you can use with `expo-sqlite`.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/sqlite-next.mdx:348]:
```suggestion
The `expo-sqlite` library is designed to be a solid SQLite foundation. It enables broader integrations with third-party libraries for more advanced higher-level features. Here are some of the libraries that you can use with `expo-sqlite`.
```

File: docs/pages/versions/unversioned/sdk/sqlite-next.mdx [hunk 3]
```diff
@@ -343,6 +343,23 @@ const row = await db.getFirstAsync<{ data: Uint8Array }>('SELECT * FROM blobs');
...
+
+### Drizzle ORM
+
+[Drizzle](https://orm.drizzle.team/) is a headless TypeScript ORM with a head. Runs on Node, Bun and Deno. Lives on the Edge and yes, it's a JavaScript ORM too! It comes with a [drizzle-kit](https://orm.drizzle.team/kit-docs/overview) CLI companion for automatic SQL migrations generation and it is Expo ready.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/sqlite-next.mdx:352]:
```suggestion
[Drizzle](https://orm.drizzle.team/) is a ["headless TypeScript ORM with a head"](https://orm.drizzle.team/docs/overview). It runs on Node, Bun, Deno, and React Native. It also comes with a [CLI compansion called drizzle-kit](https://orm.drizzle.team/kit-docs/overview) for generating SQL migrations.
```

File: docs/pages/versions/unversioned/sdk/sqlite-next.mdx [hunk 4]
```diff
@@ -343,6 +343,23 @@ const row = await db.getFirstAsync<{ data: Uint8Array }>('SELECT * FROM blobs');
...
+
+Check out the [Drizzle ORM documentation](https://orm.drizzle.team/) and the [`expo-sqlite` integration guide](https://orm.drizzle.team/docs/get-started-sqlite#expo-sqlite) for more details.
+
+
+### Knex.js
+
+[Knex.js](https://knexjs.org/) is a SQL query builder that is flexible, portable, and fun to use!
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/sqlite-next.mdx:359]:
```suggestion
[Knex.js](https://knexjs.org/) is a SQL query builder that is ["flexible, portable, and fun to use!"](https://github.com/knex/knex)
```

**Review by brentvatne — CHANGES_REQUESTED** [review]

File: docs/pages/versions/unversioned/sdk/sqlite-next.mdx [hunk 5]
```diff
@@ -343,6 +343,23 @@ const row = await db.getFirstAsync<{ data: Uint8Array }>('SELECT * FROM blobs');
 expect(row.data).toEqual(blob);
 ```
 
+## Third-party library integrations
+
+The `expo-sqlite` library is designed to be a solid SQLite foundation. It enables broader integrations with third-party libraries for more advanced higher-level features. Here are some of the libraries that you can use with `expo-sqlite`.
+
+### Drizzle ORM
+
+[Drizzle](https://orm.drizzle.team/) is a ["headless TypeScript ORM with a head"](https://orm.drizzle.team/docs/overview). It runs on Node, Bun, Deno, and React Native. It also comes with a [CLI compansion called drizzle-kit](https://orm.drizzle.team/kit-docs/overview) for generating SQL migrations.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/sdk/sqlite-next.mdx:352]:
```suggestion
[Drizzle](https://orm.drizzle.team/) is a ["headless TypeScript ORM with a head"](https://orm.drizzle.team/docs/overview). It runs on Node.js, Bun, Deno, and React Native. It also has a CLI compansion called [`drizzle-kit`](https://orm.drizzle.team/kit-docs/overview) for generating SQL migrations.
```

Let's highlight the CLI tool/package name with inline code highlight and for link, only highlight the library name as well.

**Review by amandeepmittal — APPROVED** [review]

**Pull request merged**